tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -826,7 +834,7 @@ where
 				signer,
 				who,
 				assertion,
-				vc_payload,
+				(vc_payload, vc_logs),
```

Comment:
I'm a bit worried about submitting logs to parentchain. I would suggest to include them in rpc response only.